### PR TITLE
Reference ol.MapBrowserEvent instead of ol.MapBrowserPointerEvent

### LINKED ISSUE #5414: MapBrowserPointerEvent not in api docs
For example, http://openlayers.org/en/master/apidoc/ol.interaction.ModifyEvent.html points to http://openlayers.org/en/master/apidoc/ol.MapBrowserPointerEvent.html, but this page does not exist.

Am I right in thinking that `oli.MapBrowserPointerEvent` needs to be added to `externs/oli.js` and that `ol.MapBrowserPointerEvent` needs to implement it?

### PR DESCRIPTION
**Pull request by ahocevar**
There is nothing exportable in ol.MapBrowserEvent in addition to what is exportable in ol.MapBrowserEvent.

Fixes #5414.

### CODE REVIEW

File: changelog/upgrade-notes.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Upgrade notes
 
+#### `ol.interaction.ModifyEvent` changes
+
+The event object previously had a `mapBrowserPointerEvent` property, which has bewen renamed to `mapBrowserEvent`.
```
Comment (probins) [inline changelog/upgrade-notes.md]: `map` should read `Map`. 'been renamed'.

**Comment by probins** [comment]
MapBrowserPointerEvent is also referred to in http://openlayers.org/en/master/apidoc/ol.events.condition.html (mouseOnly). That seems to be the only reference in the apidocs apart from interaction.Modify

File: changelog/upgrade-notes.md (hunk 1 above)
Comment (ahocevar) [inline changelog/upgrade-notes.md]: The former isn't true, and I fixed the latter.

**Comment by ahocevar** [comment]
I removed that too now. No more `ol.MapBrowserPointerEvent` in the API.

**Comment by fredj** [comment]
LGTM

**Pull request merged**